tests/profiles/athena/test_athena_access_key.py

```diff
@@ -100,15 +146,20 @@ def test_athena_profile_args(
     """
     Tests that the profile values get set correctly for Athena.
     """
+    with patch(
+        "cosmos.profiles.athena.access_key.AthenaAccessKeyProfileMapping.get_temporary_credentials",
+        return_value=mock_assumed_credentials,
+    ):
+        yield
     profile_mapping = get_automatic_profile_mapping(
         mock_athena_conn.conn_id,
     )
 
     assert profile_mapping.profile == {
         "type": "athena",
-        "aws_access_key_id": mock_athena_conn.login,
-        "aws_secret_access_key": "{{ env_var('COSMOS_CONN_AWS_AWS_SECRET_ACCESS_KEY') }}",
-        "aws_session_token": "{{ env_var('COSMOS_CONN_AWS_AWS_SESSION_TOKEN') }}",
+        "aws_access_key_id": mock_assumed_credentials.access_key,
+        "aws_secret_access_key": mock_assumed_credentials.secret_key,
+        "aws_session_token": mock_assumed_credentials.token,
```

Comment:
Before, we were obfuscating the secrets and setting them in environment variables. This behaviour should continue for temporary credentials as well.